front/lib/api/workspace.ts

```diff
@@ -107,9 +103,11 @@ export async function getMembers(
   auth: Authenticator,
   {
     roles,
+    activeOnly,
```

Comment:
I was unsure if I wanted to make this function part of the resource, but decided against it since `members` is not exactly the same as memberships (joining memberships, but also users and workspaces). 
So IMO, this will rather be part of the `Workspace` resource once it lands.

I also added support for getting active members only.